users/views.py

```diff
@@ -1,3 +1,35 @@
-from django.shortcuts import render
+import jwt
+import requests
 
-# Create your views here.
+from django.views import View
+from users.models import User
+from django.http  import JsonResponse
+from CSS.settings import SECRET_KEY, ALGORITHM
+from datetime     import datetime, timedelta
+
+class KakaoSignIn(View):
+    def post(self, request):
+        try:
+            access_token       = request.headers.get('Authorization')
+            kakao_info_api     = 'https://kapi.kakao.com/v2/user/me'
+            headers            = {'Authorization':f'Bearer ${access_token}'}
+            user_info_response = requests.get(kakao_info_api,headers=headers, timeout=5)
```

Comment:
```suggestion
            response = requests.get(kakao_info_api,headers=headers, timeout=5)
```
1. requests를 사용할 때 timeout 부여해 주신 점 잘 하셨습니다.
2. requests를 여러번 사용하는 것이 아니므로 그냥 response로도 충분할 것 같습니다.
3. 되도록 info처럼 줄임말 사용은 자제해 주세요.

Reply:
넵